tests-integration/Cargo.toml

```diff
@@ -0,0 +1,62 @@
+[package]
+name = "tests-integration"
+version = "0.1.0"
+authors = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+repository = { workspace = true }
+readme = "README.md"
+keywords = [ "blockchain", "consensus", "cosmos", "ibc", "tendermint" ]
+publish = false
+description = """
```

Comment:
Just being pedantic. I know it's also true for other cargo manifest files. The [cargo-book](https://doc.rust-lang.org/cargo/reference/manifest.html#the-description-field) mentions `description` to be a short blurb. I think, we should avoid multiline descriptions here. The crates-io shows the `readme` anyway.

Reply:
That's fair. But yea, this is more applicable to the other ibc-rs crates that are published; many of them have multi-line descriptions in their manifests.